### PR TITLE
Pinecone v2 destination connector

### PR DESCRIPTION
**Pull request by Paul-Cornell**
See:

- API: https://unstructured-53-docs-36-pinecone.mintlify.app/api-reference/ingest/destination-connector/pinecone
- Open source: https://unstructured-53-docs-36-pinecone.mintlify.app/open-source/ingest/destination-connectors/pinecone

### CODE REVIEW

File: snippets/general-shared-text/pinecone.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The Pinecone prerequisites:
```
Comment (MKhalusova) [inline snippets/general-shared-text/pinecone.mdx:1]: This snippet doesn't show up on staging.

File: snippets/general-shared-text/pinecone-cli-api.mdx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+<AdditionalIngestDependencies />
+
+The following environment variables:
+
+- `PINECONE_API_KEY` - The Pinecone API, represented by `--api-key` (CLI) or `api_key` (Python, in the `PineconeAccessConfig` object).
+- `PINECONE_INDEX_NAME` - The Pinecone serverless index name, represented by `--index-name` (CLI) or `index_name` (Python).
+- `PINECONE_ENVIRONMENT_NAME` - The Pinecone environment name, represented by `--environment` (CLI) or `environment` (Python).
+
+<Note>As of January 2024, a Pinecone environment name is no longer required. However, the connector still requires it. You can set `PINECONE_ENVIRONMENT_NAME` to an empty string.</Note>
```
Comment (MKhalusova) [inline snippets/general-shared-text/pinecone-cli-api.mdx:17]: This looks like an issue that needs to be addressed. 
We can keep the note in the documentation until it's resolved, but it should be fixed in the ingest library eventually.

Comment (MKhalusova) [reply snippets/general-shared-text/pinecone-cli-api.mdx:17]: Can you log an issue in the ingest repo please?

**Review by MKhalusova — CHANGES_REQUESTED** [review]

File: snippets/general-shared-text/pinecone.mdx (hunk 1 above)
Comment (Paul-Cornell) [reply snippets/general-shared-text/pinecone.mdx:1]: It works fine for me when I run `mintlify dev` locally. 

Would you be willing to pull this branch locally and run `mintlify dev` to confirm?

I can ping the Mintlify team to have them weigh in on this as well.

Comment (Paul-Cornell) [reply snippets/general-shared-text/pinecone.mdx:1]: I just checked again, and they look as expected. Could you try checking again?

**Review by MKhalusova — APPROVED** [review]
Looks good now. Not sure why it wasn't displaying before

**Pull request merged**